<launch>

  <!-- arm manipulation actionlib server -->
  <node pkg="srs_assisted_arm_navigation" name="but_arm_manipulation_node" type="but_arm_manipulation_node" respawn="false" output="screen">
    <param name="start_timeout" type="double" value="60.0"/>
    <param name="solve_timeout" type="double" value="300.0"/>
    <param name="inflate_bb" type="double" value="1.1"/>
  </node>
  
</launch>
